cmd/ibctest/ibctest_test.go

```diff
@@ -258,6 +259,11 @@ func subcommand() string {
 }
 
 func runDebugTerminalUI(ctx context.Context) error {
+	// TUI has features that copy text to your clipboard.
+	if err := clipboard.Init(); err != nil {
```

Comment:
I don't like the hard dependency on CGo for clipboard support -- this tends to show up once you're shipping a binary to an Alpine system, using musl instead of glibc, and you need to start jumping through hoops to get the binary to run there.

Reply:
I'm with you and great pushback. CGO 👎.  Like I originally planned, I'll swap out the clipboard library and handle when clipboard isn't available.